### PR TITLE
Feature/changes

### PR DESCRIPTION
**Pull request by rem1niscence**
• Add show/hide table (don't show icon when there's no data) 
• Add scroll to data only on tables
• Add + 1 to index
• Create ToC/Privacy Policy page (empty)

### CODE REVIEW

File: frontend/src/_helpers.js [hunk 1]
```diff
@@ -131,3 +134,50 @@ export const scrollToId = (id) => {
...
+      dataField: "ticker",
+      text: "Ticker",
+    },
+    {
+      dataField: "index",
+      text: "Index",
+      headerFormatter: showFormatter,
+    },
+  ];
+};
```
Comment (xerp) [inline frontend/src/_helpers.js]: The requirement is not only for networks, if for all tables.

**Review by xerp — CHANGES_REQUESTED** [review]

File: frontend/src/views/Apps/AppPassphrase/AppPassphrase.js [hunk 2]
```diff
@@ -319,8 +319,9 @@ class AppPassphrase extends Component {
         <Row className="mb-5 app-networks">
           <Col>
             <Segment label="Networks">
-              <BootstrapTable
-                classes="table app-table app-table-empty table-striped"
+              <AppTable
+                scroll
+                toggler={[].length > 0}
```
Comment (xerp) [inline frontend/src/views/Apps/AppPassphrase/AppPassphrase.js]: Missing **app-table-empty** css class. Why?

File: frontend/src/views/Apps/AppsMain/AppsMain.js [hunk 3]
```diff
@@ -268,16 +268,17 @@ class AppsMain extends Main {
...
                 loadMore={this.loadMoreRegisteredApps}
                 useWindow={false}
                 hasMore={hasMoreRegisteredItems}
                 loader={loader}
               >
-                <BootstrapTable
-                  classes={`app-table ${registeredItems.length === 0 ? "app-table-empty" : null}`}
+                <AppTable
+                  scroll
+                  toggle={registeredItems.length > 0}
```
Comment (xerp) [inline frontend/src/views/Apps/AppsMain/AppsMain.js:281]: Missing **app-table-empty** css class. Why?

File: frontend/src/views/Apps/Import/ImportApp.js [hunk 4]
```diff
@@ -297,8 +297,9 @@ class Import extends Component {
         <Row className="mt-2 app-networks">
           <Col className="title-page">
             <h3>Networks</h3>
-            <BootstrapTable
-              classes="table app-table app-table-empty table-striped"
+            <AppTable
+              scroll
+              toggle={[].length > 0}
```
Comment (xerp) [inline frontend/src/views/Apps/Import/ImportApp.js]: Missing **app-table-empty** css class. Why?

File: frontend/src/views/Dashboard/Dashboard.js [hunk 5]
```diff
@@ -200,10 +196,9 @@ class Dashboard extends Component {
               }`}
             >
               <Segment scroll={false} label="Registered Nodes">
-                <BootstrapTable
-                  classes={`app-table scroll ${
-                    userNodes.length === 0 ? "app-table-empty" : null
-                  }`}
+                <AppTable
+                  scroll
+                  toggle={userNodes.length > 0}
```
Comment (xerp) [inline frontend/src/views/Dashboard/Dashboard.js:201]: Missing **app-table-empty** css class. Why?

File: frontend/src/views/Dashboard/Dashboard.js [hunk 6]
```diff
@@ -220,11 +215,10 @@ class Dashboard extends Component {
...
               <Segment scroll={false} label="Registered Apps">
-                <BootstrapTable
-                  classes={`app-table scroll ${
-                    userApps.length === 0 ? "app-table-empty" : null
-                  }`}
-                  keyField="pocketApplication.publicPocketAccount.address"
+                                <AppTable
+                  scroll
+                  toggle={userApps.length > 0}
+                  keyField="pocketNode.publicPocketAccount.address"
```
Comment (xerp) [inline frontend/src/views/Dashboard/Dashboard.js:221]: Missing **app-table-empty** css class. Why?

File: frontend/src/views/Dashboard/Dashboard.js [hunk 7]
```diff
@@ -242,17 +236,12 @@ class Dashboard extends Component {
               }`}
             >
               <Segment scroll={false} label="Supported Blockchains">
-                <BootstrapTable
-                  classes={`app-table scroll ${hideSupport ? "hide" : ""}`}
+                <AppTable
```
Comment (xerp) [inline frontend/src/views/Dashboard/Dashboard.js:239]: Missing **app-table-empty** css class. Why?

File: frontend/src/views/Dashboard/Dashboard.js [hunk 8]
```diff
@@ -266,17 +255,12 @@ class Dashboard extends Component {
               }`}
             >
               <Segment scroll={false} label="Most popular chains">
-                <BootstrapTable
-                  classes={`app-table scroll ${hideMostPopular ? "hide" : ""}`}
+                <AppTable
```
Comment (xerp) [inline frontend/src/views/Dashboard/Dashboard.js:258]: Missing **app-table-empty** css class. Why?

**Review by xerp — CHANGES_REQUESTED** [review]

File: frontend/src/views/Apps/AppPassphrase/AppPassphrase.js (hunk 2 above)
Comment (rem1niscence) [reply frontend/src/views/Apps/AppPassphrase/AppPassphrase.js]: I encapsulated all of that logic on the `<AppTable>` component

Comment (xerp) [reply frontend/src/views/Apps/AppPassphrase/AppPassphrase.js]: Good

File: frontend/src/views/Apps/Import/ImportApp.js (hunk 4 above)
Comment (xerp) [reply frontend/src/views/Apps/Import/ImportApp.js]: In this case, this table is always empty, so, by default we don't need to expand/collapse.

**Review by xerp — APPROVED** [review]

**Pull request merged**